catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -200,5 +200,99 @@
+
+    <bean id="emailNotifierService"
+          class="org.codice.ddf.catalog.ui.query.monitor.email.EmailNotifier">
+
+        <cm:managed-properties
+                persistent-id="org.codice.ddf.catalog.ui.query.monitor.email.EmailNotifier"
+                update-strategy="container-managed"/>
+
+        <argument
+                value="The workspace '%[attribute=title]' (id: %[attribute=id]) contains up to %[hitCount] query hits."/>
+        <argument value="%[attribute=title] (id: %[attribute=id]): workspace query notice"/>
+        <argument value="donotreply@test.com"/>
+        <argument value="connexta-com.mail.protection.outlook.com"/>
+        <argument>
+            <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.ListMetacardFormatter">
+                <argument>
+                    <list>
+                        <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.AttributeMetacardFormatter">
+                            <argument value="N/A"/>
+                        </bean>
+                        <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.HitCountFormatter"/>
+                    </list>
+                </argument>
+            </bean>
+        </argument>
+        <!-- TODO remove TrivialSubscriptionsPersistenceStore after testing -->
+        <argument>
+            <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.TrivialSubscriptionsPersistentStore">
+                <argument value="glen.hein@connexta.com"/>
+            </bean>
+        </argument>
+        <!-- TODO use this after testing: <argument ref="workspacePersistentStore"/> -->
+    </bean>
+
+    <bean id="workspaceQueryService"
+          class="org.codice.ddf.catalog.ui.query.monitor.impl.WorkspaceQueryService">
+
+        <cm:managed-properties
+                persistent-id="org.codice.ddf.catalog.ui.query.monitor.impl.WorkspaceQueryService"
+                update-strategy="container-managed"/>
+
+        <argument>
+            <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.QueryUpdateSubscriberList">
+                <argument>
+                    <list>
+                        <ref component-id="emailNotifierService"/>
+                    </list>
+                </argument>
+            </bean>
+        </argument>
+        <argument ref="queryWorkspaceService"/>
+        <argument ref="catalogFramework"/>
+        <argument ref="filterBuilder"/>
+        <argument>
+            <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.quartz.DefaultScheduler"/>
+        </argument>
+        <argument>
+            <!-- TODO Change to org.codice.ddf.catalog.ui.query.monitor.impl.quartz.DailyAtMidnight for final production -->
+            <bean class="org.codice.ddf.catalog.ui.query.monitor.impl.quartz.CronString">
+                <argument value="0 * * * * ?"/>
```

Comment:
Make this configurable.  You can remove DailyAtMidnight and just use the cron string.